packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -205,6 +221,14 @@ export function PanelChrome({
       </div>
     </div>
   );
+
+  return collapsible ? (
+    <Collapse label={title} collapsible isOpen={open} onToggle={() => toggle()} collapseBodyClassName={styles.collapse}>
+      {chromeContent}
+    </Collapse>
+  ) : (
```

Comment:
I don't think we can use Collapse here, won't this lead to double titles/headings?